docs/en/connector-v2/sink/Hive.md

```diff
@@ -25,21 +25,24 @@ By default, we use 2PC commit to ensure `exactly-once`
   - [x] parquet
   - [x] orc
   - [x] json
+  - [x] hive_text
```

Comment:
Hi lightzhao, Should we document what's the differences between text and hive_text?

Reply:
hive_text is compatible with the text storage format of hive collection data type, array, map, struct.

Reply:
I think you should metion it on the doc.

Reply:
ok